test/spec/index-spec.js

```diff
@@ -2,7 +2,7 @@ const { expect } = require('chai');
 
 const {
   Linter
-} = require('bpmnlint');
+} = require('../../lib/index');
```

Comment:
Hmm, any reason you did this change? The tests should verify that we the package export the right entry point, from the package import.

If we remove this, then no one is verifying that external libraries can `const { Linter } = require('bpmnlint')` or `import { Linter } from 'bpmnlint'`.